src/lib/components/linetemporalchart/LineTemporalChart.component.js

```diff
@@ -593,7 +593,7 @@ function LineTemporalChart({
         <Legends>
           {legendLabels.map((legend, index) => {
             return (
-              <>
+              <div key={index}>
```

Comment:
nit: You don't need to introduce an extra div to add a key here. You can use `<React.Fragment key={index}>`.